IoTCoreDefaultApp/IoTCoreDefaultApp/Strings/de-DE/Resources.resw

```diff
@@ -635,11 +635,11 @@
     <comment>Format string for message displayed when Inbound pairing handler registration already done</comment>
   </data>
   <data name="BluetoothInboundRegistrationFailed" xml:space="preserve">
-    <value>Fehler beim Registrieren für eingehende Kopplung</value>
+    <value>Fehler beim Registrieren für eingehende Kopplung.</value>
     <comment>Message displayed when Inbound pairing handler registration fails</comment>
   </data>
   <data name="BluetoothInboundRegistrationSucceededFormat" xml:space="preserve">
-    <value>Registrierung für eingehende Kopplung war erfolgreich, unterstützte Zeremonien: {0}</value>
+    <value>Die Registrierung für die eingehende Kopplung war erfolgreich. Unterstützte Zeremonien: {0}.</value>
```

Comment:
String in English: <value>Successfully registered for inbound pairing, supported ceremonies: {0} . </value>  
Note that the (,) is replaced by a (.)